ledger/participant-integration-api/src/test/lib/scala/platform/indexer/ha/IndexerStabilityTestFixture.scala

```diff
@@ -94,12 +107,34 @@ object IndexerStabilityTestFixture {
                     .acquire()
                   metricRegistry = new MetricRegistry
                   metrics = new Metrics(metricRegistry)
+                  updateInMemoryBuffersFlow <- ledgerApiUpdateFlow(
+                    metrics,
+                    servicesExecutionContext,
+                  )
+                  dbType = DbType.jdbcType(jdbcUrl)
+                  storageBackendFactory = StorageBackendFactory.of(dbType)
+
+                  stringInterningStorageBackend =
+                    storageBackendFactory.createStringInterningStorageBackend
                   // Create an indexer and immediately start it
                   indexing <- new StandaloneIndexerServer(
                     readService = readService,
                     config = indexerConfig,
                     metrics = metrics,
                     lfValueTranslationCache = LfValueTranslationCache.Cache.none,
+                    stringInterningView = new StringInterningView(
+                      loadPrefixedEntries = (fromExclusive, toInclusive, dbDispatcher) =>
+                        implicit loggingContext =>
+                          dbDispatcher.executeSql(
+                            metrics.daml.index.db.loadStringInterningEntries
+                          ) {
+                            stringInterningStorageBackend.loadStringInterningEntries(
+                              fromExclusive,
+                              toInclusive,
+                            )
+                          }
+                    ),
+                    updateInMemoryBuffersFlow = updateInMemoryBuffersFlow,
```

Comment:
Adds in-memory fan-out and mutable contract state caches load to the indexer under benchmark